modules/ROOT/pages/api-experience-hub-lightning-components.adoc

```diff
@@ -1,57 +1,104 @@
-= API Experience Hub Lightning Components 
+= API Experience Hub Lightning Web Components 
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
 
-You can design the pages of your API Experience Hub portal by adding and configuring Lightning components in the Builder.
+You can design the pages of your API Experience Hub portal by adding and configuring Lightning web components in the Builder.
```

Comment:
Just checking that Lightning web components is not a proper noun. If it is, then it probably should be init cap (LWC).

Reply:
https://developer.salesforce.com/docs/platform/lwc/guide/get-started-release-notes.html

Reply:
Global: Review the short descriptions to ensure you've covered the benefits.

Reply:
I double-checked. It should be Lightning components since Lightning Web components refers to the programming model.